From 66a640f5169cd09fb428b56f0ab9efb028e8ca77 Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Sun, 4 Jul 2021 19:05:43 -0300
Subject: [PATCH] Local qscintilla2_qt5

---
 QtiASL.pro | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/QtiASL.pro b/QtiASL.pro
index 4362f56..f76d494 100644
--- a/QtiASL.pro
+++ b/QtiASL.pro
@@ -22,7 +22,7 @@ LIBS += \
 
 unix:!macx:{
 
-    LIBS += -L$PWD libqscintilla2_qt5.so
+    LIBS += -lqscintilla2_qt5
 
     QMAKE_RPATHDIR=$ORIGIN
     QMAKE_LFLAGS += -no-pie
@@ -131,7 +131,7 @@ FORMS += \
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = /usr/bin
 !isEmpty(target.path): INSTALLS += target
 
 DISTFILES += \
-- 
2.32.0

